Source/WarpX.cpp

```diff
@@ -555,6 +624,21 @@ WarpX::ReadParameters ()
             }
         }
 
+        std::vector<std::string> signals_in;
+        pp_warpx.queryarr("break_signals", signals_in);
```

Comment:
Please don't forget to add these new input options to `Docs/source/usage/parameters.rst`